src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AdjudicationsResourceTest.java

```diff
@@ -123,6 +195,55 @@ public void returns403IfInvalidRole() {
         }
     }
 
+    @Nested
+    public class CreateAdjudicationWithoutCreationRequestData {
+
+        @Test
+        public void returnsExpectedValue() {
+            final var token = validToken(List.of("ROLE_MAINTAIN_ADJUDICATIONS"));
+            final var body = Map.of(
+                "offenderNo", "A1234AE",
+                "agencyId", "MDI",
+                "incidentTime", "2021-01-04T10:12:44",
+                "incidentLocationId", -31L,
+                "statement", "Example statement");
+
+            final var httpEntity = createHttpEntity(token, body);
+
+            final var response = testRestTemplate.exchange(
```

Comment:
For future tests might be worth using webtestclient and nice inline json assets rather than that awful assertThatJsonFileAndStatus

Reply:
Yep - I may well fix that in the next PR, which will be necessary when the client is fixed.